nnpdf_data/nnpdf_data/new_commondata/STAR-2006_SHP_200GEV/uncertainties.yaml

```diff
@@ -0,0 +1,22 @@
+definitions:
+  stat:
+    description: Statistical uncertainty
+    treatment: ADD
+    type: UNCORR
+  syst:
```

Comment:
From https://arxiv.org/pdf/1309.1800 fig 7, I think we need to add 6% polarisation beam uncertainty: CORR, STAR2006POL.

Reply:
Upon first reading the caption and the corresponding text, I understood that the 6% scaling was already included in the vertical bar and hence in the overall systematics. But you might be right that this is not the case.

It is a bit annoying that these experimental papers are not always clear in stating what they did.

Reply:
Indeed this is not super clear. For instance in the first bin If I do 0.06*0.023=0.00138, while the quoted syst is 0.02. 
So I'm not sure the paper says:

```
Uncertainty in the product of beam polarizations results in a 6% vertical scale uncertainty as
indicated in the figure.
```

Reply:
I included the 6% beam polarization in any case - better be conservative about them if we are not sure.